sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -70,8 +70,53 @@ APIs for using IotHub v2 jobs
 </details>
 
 <details><summary><b>Messages</b></summary>
-Feedback messages, sending cloud to device messages (missing from current swagger), and purging cloud to device message queue
+Ssending cloud to device messages, receiving feedback messages, and purging cloud to device message queue
+
 ```csharp
+public class CloudToDeviceMessages
+{
+    /// <summary>
+    /// Send cloud to device message for a device.
+    /// </summary>
+    /// <param name="deviceId">The unique identifier of the device.</param>
+    /// <param name="message">The cloud to device message to be sent to the device. </param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The Http response.</returns>
+    public virtual async Task<Response> SendMessageAsync(String deviceId, CloudToDeviceMessage message, CancellationToken cancellationToken = default) { }
+
+    /// <summary>
+    /// Retrieve feedback notification for cloud to device messages.
+    /// </summary>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The message feedback batch, containing the details of the final state of sent messages.</returns>
+    public virtual async Task<Response<MessageFeedbackBatch>> GetMessageFeedbackAsync(CancellationToken cancellationToken = default) { }
+
+    /// <summary>
+    /// Complete a cloud to device feedback message. A completed message is deleted from the service's feedback queue.
+    /// </summary>
+    /// <param name="lockToken">The lock token obtained when the cloud to device message was received, and provided to resolve race conditions when completing a feedback message.
+    /// TODO: lockToken is from the C2D message received on device or from message feedback received on service client?</param>
```

Comment:
From the track 1 SDK, it looks like it is the lockToken received on the message feedback batch, but surprisingly the MS doc doesn't mention a lockToken to be a part of the feedback batch: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2d#message-feedback

Need confirmation from the service team.